cockatrice/src/user_context_menu.cpp

```diff
@@ -89,6 +94,27 @@ void UserContextMenu::banUser_processUserInfoResponse(const Response &r)
     dlg->show();
 }
 
+void UserContextMenu::adjustMod_processUserResponse(const Response &resp, const CommandContainer &commandContainer)
```

Comment:
You joke, but yes, I've done this. Bad connections exist

On Thu, Aug 20, 2015, 9:55 PM woogerboy21 <notifications@github.com> wrote:

> In cockatrice/src/user_context_menu.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/1391#discussion_r37600346>:
>
> > @@ -89,6 +94,27 @@ void UserContextMenu::banUser_processUserInfoResponse(const Response &r)
> >      dlg->show();
> >  }
> >
> > +void UserContextMenu::adjustMod_processUserResponse(const Response &resp, const CommandContainer &commandContainer)
>
> The user menu is were its fired from, why wouldnt the response be
> processed from it as well? You quicker than lightning and going to close
> all your context menus before the server has time to process the command?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1391/files#r37600346>.
>